docs/user-guide/mvd-editor.md

```diff
@@ -2,40 +2,64 @@
 
 With the Zowe Editor, you can create and edit the many types of files.
 
-## Specifying a language server
+## Specifying a highlighting language
 
-To specify a language server, complete these steps:
+1. Click **Language** on the editor menu bar. A dropdown menu will be displayed.
+2. From the dropdown, select the desired language. Plain Text will be chosen by default if the automatic language detection is not able to determine the language. 
 
-1. From the **Language Server** menu, select **URL***.
-2. From the **Language Server Setting**, **Put your config here** area, paste your configuration.
-3. Ensure that the **Enable Language Server** check box is selected.
-4. Click **Save**.
+## Open a dataset
 
-## Specifying a language
+To open a dataset, follow these steps:
 
-From the **Language** menu, select the language you want to use.
+1. From the **File** menu, select Open Datasets. You can also use (ALT+K).
+2. In the Dataset field, specify the name of the dataset you want to open. 
+3. Click **Open** 
+
+## Deleting a dataset
+
+1. In the file tree, right-click on a dataset you want to delete.
+2. From the right-click menu, click **Delete**. A warning dialogue will appear. 
+3. Click **Delete** 
```

Comment:
Oops, I totally missed this. This whole section should say "Deleting a file or folder" We currently cannot delete datasets, but we do have deletion support for files & folders